crates/next-custom-transforms/src/chain_transforms.rs

```diff
@@ -307,7 +314,11 @@ where
             crate::transforms::debug_fn_name::debug_fn_name(),
             opts.debug_function_name
         ),
-        as_folder(crate::transforms::pure::pure_magic(comments)),
+        as_folder(crate::transforms::pure::pure_magic(comments.clone())),
+        Optional::new(
+            linter(lint_error_comment(comments)),
+            opts.lint_error_comments
```

Comment:
Is this name ideal?

Reply:
`lint_codemod_comments` might be more correct